src/it/java/io/deephaven/benchmark/tests/standard/formula/InlineFormulaTest.java

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2022-2023 Deephaven Data Labs and Patent Pending */
+package io.deephaven.benchmark.tests.standard.formula;
+
+import org.junit.jupiter.api.*;
+import io.deephaven.benchmark.tests.standard.StandardTestRunner;
+
+/**
+ * Standard tests for running basic formulas (not functions) inline for select, update, view, update_view, etc. These
+ * tests aggregate the resulting values to make sure the formulas are actually run (as in the case of views). Theses
+ * tests are meant to be compared, and so use the same data.
+ */
+public class InlineFormulaTest {
```

Comment:
Certainly seems cleaner and less duplicative, but haven't you dropped several cases in this transformation?

Reply:
The purpose of those tests was to test the difference between doing the same inline formulas in update, update_view, select, and view.  Doing the _2 Calc 2 Col_ tests seemed sufficient.  The ones I removed were _1 Calc 2 Col_ tests and _int-vs-floats_.  Would those tests shed light on the difference between the operations?